cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControl.java

```diff
@@ -319,6 +319,16 @@ public void addRecentBrokersPermanently(Set<Integer> brokersToAdd, boolean isRem
     return isRemoved ? _executor.recentlyRemovedBrokers() : _executor.recentlyDemotedBrokers();
   }
 
+  /**
+   * Dynamically set the interval between checking and updating (if needed) the progress of an initiated execution.
+   *
+   * @param requestedExecutionProgressCheckIntervalMs The interval between checking and updating the progress of an initiated
+   *                                                 execution (if null, use {@link Executor#MIN_EXECUTION_PROGRESS_CHECK_INTERVAL_MS}).
```

Comment:
if requested is null, looks like we are using `_defaultExecutionProgressCheckIntervalMs` instead of `MIN_EXECUTION_PROGRESS_CHECK_INTERVAL_MS`